### DISCUSSION TITLE
#map goto matching room description

### DISCUSSION THREAD

**Discussion by electrobassoon** [scandum/tintin#71] [Script help]
Hello,
I can't seem to figure out how to have #map goto to match my room description. I copied a script more or less from here https://tintin.sourceforge.io/forum/viewtopic.php?t=1684 that grabs the room name and description when I enter, and then I updated the room with it. It works to goto the current roomname, but unfortunately there are lots of rooms with the same name and I want to use the description to help find the right one. It seems like the documentation says it should be possible. Maybe I just don't understand the syntax? The exits also change depending on which doors are open, so I'd like to skip that argument.

**Comment by scandum** [comment]
Your problem is likely with the use of `\n` in the example you linked. Probably easiest to use a space instead for multi-line descriptions.

Might be that script once worked, but room description searches are regex enabled currently, so \n searches for a newline character.
